fsm-calculator/src/main/java/org/egov/fsm/calculator/web/models/AuditDetails.java

```diff
@@ -0,0 +1,157 @@
+package org.egov.fsm.calculator.web.models;
+
+import java.util.Objects;
+
+import org.springframework.validation.annotation.Validated;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
+
+/**
+ * Collection of audit related fields used by most models
+ */
+@ApiModel(description = "Collection of audit related fields used by most models")
+@Validated
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2020-06-23T05:54:07.373Z[GMT]")
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class AuditDetails   {
+  @JsonProperty("createdBy")
+  private String createdBy = null;
+
+  @JsonProperty("lastModifiedBy")
+  private String lastModifiedBy = null;
+
+  @JsonProperty("createdTime")
+  private Long createdTime = null;
+
+  @JsonProperty("lastModifiedTime")
+  private Long lastModifiedTime = null;
+
+  public AuditDetails createdBy(String createdBy) {
+    this.createdBy = createdBy;
+    return this;
+  }
+
+  /**
+   * username (preferred) or userid of the user that created the object
+   * @return createdBy
+  **/
+  @ApiModelProperty(value = "username (preferred) or userid of the user that created the object")
+  
+    public String getCreatedBy() {
+    return createdBy;
+  }
+
+  public void setCreatedBy(String createdBy) {
```

Comment:
use lombok annotation @Getter anf @Setter in all POJO's

Reply:
these models are generated from swagger. with the suggestion from Ghanshyam on BPA.
same has been followed here and i have added NoArg, allArg and Building annotations to the models.

Reply:
cleanup the getter and setter as we move forward.